app/templates/_react.html

```diff
@@ -11,6 +17,10 @@
 {% block body_end %}
   <script type="text/javascript" src="{{ component.get_vendor_bundle() }}" defer></script>
   <script type="text/javascript" src="{{ component.get_bundle() }}" defer></script>
+  {# May not be the best place to load this, time will tell. #}
```

Comment:
we also use profiles in admin so this will need to be copied sooner rather than later.